### PR TITLE
Update FOP to 2.9

### LINKED ISSUE #4272: Update Apache FOP to version 2.9
A new FOP version is available: https://xmlgraphics.apache.org/fop/2.9/releaseNotes_2.9.html
With all its satellite libraries:
 - Apache XML Graphics Commons
 - Apache FOP-PDF-Images
 - Apache Batik (1.17)

Also in `org.dita.pdf2.fop/lib` there's an extra fop.jar which seems useless.

### PR DESCRIPTION
**Pull request by jelovirt**
## Description
Update FOP and related libraries to version 2.9.

## Motivation and Context
Fixes #4272

## How Has This Been Tested?
Existing tests pass and manual testing.

## Type of Changes
- New feature _(non-breaking change which adds functionality)_

### CODE REVIEW

File: src/main/config/logback.xml
Comment (raducoravu) [inline src/main/config/logback.xml:1]: I do not understand what this line is for.
There are still errors like:

     [ERROR] Unsupported TrueType font: No Unicode or Symbol cmap table not present. Aborting

reported in the console when running the DITA OT on my mac. If you want such lines to disappear which makes sense you could maybe turn it off:

     <logger name="org.apache.fop.fonts.truetype.TTFFile" level="OFF"/>

**Review by raducoravu — CHANGES_REQUESTED** [review]

File: src/main/config/logback.xml
Comment (jelovirt) [reply src/main/config/logback.xml:1]: @raducoravu if you remove the configuration and run `dita` CLI, you notice what messages the configuration removes. That error should not be ignored.

Comment (raducoravu) [reply src/main/config/logback.xml:1]: I tried this but I do not notice what messages the configuration removes. There are usually lots of messages when fonts are loaded and cached by Apache FOP on the first run. Once they are cached on subsequent runs I do not have any extra message that the line you added would hide.

**Review by raducoravu — APPROVED** [review]

File: src/main/config/logback.xml
Comment (jelovirt) [reply src/main/config/logback.xml:1]: I can't reproduce the log messages now because I don't remember how I got them.

**Pull request merged**